### ISSUE TITLE
Clicking View Seller instead of buy

### ISSUE THREAD

**Issue by Kam3l1** [Scruffydrew/Forza-Horizon-5-Auction-House-Sniper#22]
It keeps clicking the view seller instead of buy.

**Comment by 0XenoX0** [comment]
Yes actually i have the exact same problem. but you can't blame because of this he already is an amazing person for making us a full script. We need help if you can arrange the script please so we can snipe cars at the latest update of FH5.
Thanks You.

**Comment by ihackeverything** [comment]
> Yes actually i have the exact same problem. but you can't blame because of this he already is an amazing person for making us a full script. We need help if you can arrange the script please so we can snipe cars at the latest update of FH5. Thanks You.

works fine for me

**Comment by Kam3l1** [comment]
It is clicking view seller bcs i have less than 60fps and it is written on the page that minimum is 60fps.

**Comment by ihackeverything** [comment]
> It is clicking view seller bcs i have less than 60fps and it is written on the page that minimum is 60fps.

i sometimes ran it at under 30fps so thats not the problem

**Comment by Takata333** [comment]
Who knows how to fix it?

**Comment by Claiveur** [comment]
I actually have the same problem

**Comment by fh5sniperguy** [comment]
i have the same problem someone help
